tests/framework/helpers/HtmlTest.php

```diff
@@ -100,6 +101,68 @@ public function testJsFile()
         $this->assertEquals("<!--[if (gte IE 9)|(!IE)]><!-->\n" . '<script src="http://example.com"></script>' . "\n<!--<![endif]-->", Html::jsFile('http://example.com', ['condition' => '(gte IE 9)|(!IE)']));
     }
 
+    public function testCsrfMetaTagsDisableCsrfValidation()
+    {
+        $this->assertEquals('', Html::csrfMetaTags());
```

Comment:
It is unclear what exactly needs to be corrected. Maybe I need to rename the method `providerBeginFormSimulateViaPost` to `dataProviderBeginFormSimulateViaPost` for consistency?